src/tmux_daemon.rs

```diff
@@ -0,0 +1,38 @@
+use std::error::Error;
+use std::io::{BufRead, BufReader, Write};
+use std::process::ChildStdin;
+use std::sync::mpsc::Sender;
+use std::thread::{sleep, spawn};
+use std::time;
+
+use crate::tmux;
+
+pub fn watch_for_dead_panes(session: &str, sender: Sender<String>) -> Result<(), Box<dyn Error>> {
+    let command_mode = tmux::command_mode(session).unwrap();
```

Comment:
not sure why this child process exits almost immediately. if you look at the log you'll see some output, or maybe just %exit

Reply:
Made some progress here. The process now lives indefinitely, but I don't think the commands being sent to stdin are working correctly. The session persists after proctmux exits, and no notifications are being created.

Another bug being that text does not appear when typing to terminal stdin after proctmux exits. Maybe stdin needs to be cleaned up somehow in TmuxDaemon::kill